stdlib/SuiteSparse/src/umfpack.jl

```diff
@@ -406,6 +426,23 @@ for itype in UmfpackIndexTypes
                         mx, mz, C_NULL, lu.numeric, umf_info)
             complex(mx[], mz[])
         end
+        function logabsdet(F::UmfpackLU{T, $itype}) where {T} # return log(abs(det)) and sign(det)
+            n = checksquare(F)
+            issuccess(F) || return log(zero(real(T))), log(one(T))
```

Comment:
Is `log(one(T))` just `zero(T)`?

Reply:
Yes, I suppose it is...

Reply:
wait, it's actually not (eg `Rational`)

Reply:
After more thought, I went ahead and restricted the types to Float64 and ComplexF64. That let's us avoid the weird `log(one(T))` construction, and also matches the rest of the file, where all the methods are explicitly defined for 64-bit floats/complex floats.